content/exchange/artifacts/Windows.LOLDriverScan.yaml

```diff
@@ -0,0 +1,32 @@
+name: Windows.LOLDriverScan
+description: LolDriverScan is a golang tool that allows users to discover vulnerable drivers on their system. This tool fetches the loldrivers.io list from their APIs and scans the system for any vulnerable drivers This project is implemented in Go and does not require elevated privileges to run. https://github.com/FourCoreLabs/LolDriverScan
```

Comment:
I took a quick look at this project and there are a number of issues with integrating it with Velociraptor
- the binary goes out to the lolbin api to get the list of hashes - this can be a problem when running as a hunt (having 100k endpoints hitting the API the same time can be fun on the gateway and the provider).
- The project seems very simple - it just grabs some hashes. It looks like a full VQL equivalent is fairly simple to do. In fact it looks very similar to Windows.System.Services - can we just leverage that one?

Reply:
FWIW - I have a VQL hunt that looks at hashes in AMCache here https://github.com/mgreen27/DetectRaptor/blob/master/vql/LolDrivers.yaml
(I do need to refactor my build script to split it out a bit as the project is getting quite big nowdays)

Should be fairly easy to add in a services check too